README.md

```diff
@@ -153,76 +153,135 @@ On certain pages an API endpoint is called to receive experiment analysis data f
 
 Experimenter uses [docker](https://www.docker.com/) for all development, testing, and deployment.
 
-The following helpful commands have been provided via a Makefile:
-
-### build
+### Building
+#### make build
 
 Build the application container by executing the [build script](https://github.com/mozilla/experimenter/blob/main/scripts/build.sh)
 
-### compose_build
+#### make compose_build
 
 Build the supporting services (nginx, postgresql) defined in the [compose file](https://github.com/mozilla/experimenter/blob/main/docker-compose.yml)
 
-### up
+#### make ssl
+
+Create dummy SSL certs to use the dev server over a locally secure
+connection. This helps test client behaviour with a secure
+connection. This task is run automatically when needed.
+
+#### make kill
+
+Stop and delete all docker containers.
+WARNING: this will remove your database and all data. Use this to reset your dev environment.
+
+#### make migrate
 
-Start a dev server listening on port 80 using the [Django runserver](https://docs.djangoproject.com/en/1.10/ref/django-admin/#runserver)
+Apply all django migrations to the database.  This must be run after removing database volumes before starting a dev instance.
 
-### up_db
+#### make load_locales_countries
+
+Populates locales and countries in the database from the [Firefox Product Details package](https://pypi.org/project/django-mozilla-product-details/)
+
+#### load_dummy_experiments
```

Comment:
No `make`?

It might be fine to take `make` off all these and just note it at the top of the section since it applies to everything. Either or works

Reply:
Oh whoops missed that